main.go

```diff
@@ -157,3 +166,43 @@ func main() {
 
 	wg.Wait()
 }
+
+func getUniqueDestinations(config map[string]module) []string {
+
+	// Map filters duplicates
+	uniqueDestinationsM := make(map[string]string)
+
+	// Range over config and gather all uniquieDestinations
+	for _, m := range config {
+		if len(m.Destinations) == 0 {
+			uniqueDestinationsM[opts.ModulePath] = opts.ModulePath
+			continue
+		}
+
+		// range over Destinations and put them into map
+		for _, dst := range m.Destinations {
+			// Destination supposed to be conjunction of destination defined in file with module path
+			d := filepath.Join(dst, opts.ModulePath)
+			uniqueDestinationsM[d] = d
+		}
+	}
+
+	uniqueDestinations := make([]string, len(uniqueDestinationsM))
```

Comment:
no need to convert into a slice

you can do:
```
for dst := range uniqueDestinations {
log.Infof("[*] Removing artifacts from %s", dst)
......
}
```
directly from the map keys (since this is already a distinct list of values at this point)